docs/lib/auth/fragments/android/signin_web_ui/20_platform_specific_setup.md

```diff
@@ -16,6 +16,17 @@ your redirect URI prefix if necessary:
     </intent-filter>
 </activity>
 ```
+  
+If your app is targeting API level 30 or above, add the following queries section on the top level to `AndroidManifest.xml`.
```

Comment:
Starting next month, [Google won't even allow you to build an app that targets anything](https://developer.android.com/distribute/best-practices/develop/target-sdk) less than API 30, so I think we should just recommend that everyone do this.  Just add it to the existing snippet above, below the `HostedUIRedirectActivity` stuff.